cmd/kubeadm/app/images/images.go

```diff
@@ -64,6 +64,11 @@ func GetEtcdImage(cfg *kubeadmapi.InitConfiguration) string {
 	return GetGenericArchImage(cfg.ImageRepository, constants.Etcd, etcdImageTag)
 }
 
+// GetPauseImage returns the image for the "pause" container
+func GetPauseImage(cfg *kubeadmapi.InitConfiguration) string {
+	return GetGenericImage(cfg.ImageRepository, "pause", "3.1")
```

Comment:
I would have taken the opportunity to move the version string into a constant too.